recipe/meta.yaml

```diff
@@ -102,6 +102,7 @@ outputs:
         - python >=3.7.1
         - {{ pin_subpackage('unidist', exact=True) }}
         - mpi4py >=3.0.3
+        - mpich
```

Comment:
@YarShev `mpich` doesn't exist for windows. Will this work?

Can this be specified only for Linux systems?